Sources/Datadog/RUM/RUMMonitor/Scopes/RUMViewScope.swift

```diff
@@ -311,7 +311,7 @@ internal class RUMViewScope: RUMScope, RUMContextProvider {
                 crash: nil,
                 error: nil,
                 id: dependencies.rumUUIDGenerator.generateUnique().toRUMDataFormat,
-                loadingTime: dependencies.launchTimeProvider.launchTime?.toInt64Nanoseconds,
+                loadingTime: dependencies.launchTimeProvider.launchTime.toInt64Nanoseconds,
```

Comment:
There are two things I'm concerned about:
* The new "lazy" aspect of `.launchTime` API makes it slightly dangerous. It's easy to imagine introducing a bug by calling it somewhere earlier in the stack which will skew the measurement. Seems hard to cover this in tests.
* Using `launchTimeProvider.launchTime` to stop measurement in `RUMViewScope` skews the real measurement by the time of [`queue.async {}` used to run RUM "start view" command](https://github.com/DataDog/dd-sdk-ios/blob/a88103e71ebd79262af6723c095eb0dd1e0d83f7/Sources/Datadog/RUMMonitor.swift#L605).

What's our take on the above? Is there any alternative we can consider 🤔 💭?

Reply:
> Another approach would be to respond to the following notifications (...)

Or we could add our own notification, sent from `RUMMonitor.startView()`, but this sounds like adding cruft...

> The `launchTime` property is not lazy but computed (...). It's applying a fallback 

I totally missed that `.launchTime` is computed now. This mitigates the first point on my list 👍.

Maybe the `queue.async {}` impact is not that significant as it will be uniformly added to all measurements for the same scenario. @maxep @buranmert ?